### PR TITLE
Add post helper function for formdata

### PR DESCRIPTION
**Pull request by superfashi**
Adds a new `_CallPost` that receives a `curl_httppost` to simplify Form POST calls including log entry. This new helper function also parses the error message and prints it, like other helper functions.

### CODE REVIEW

**Comment by woswos** [comment]
Pipeline ID: 852542

**Review by woswos — APPROVED** [review]

File: src/controllerclientimpl.cpp [hunk 1]
```diff
@@ -1914,7 +1925,6 @@ void ControllerClientImpl::_UploadFileToControllerViaForm(std::istream& inputStr
         throw MUJIN_EXCEPTION_FORMAT0("failed to rewind inputStream", MEC_InvalidArguments);
     }
 
-    CURL_OPTION_SAVE_SETTER(_curl, CURLOPT_READFUNCTION, NULL, _ReadIStreamCallback);
```
Comment (ziyan) [inline src/controllerclientimpl.cpp:1917]: Where did this go?

Comment (superfashi) [reply src/controllerclientimpl.cpp:1917]: This option is not used when using httppost (as you can see other form uploads does not use this)

**Comment by ziyan** [comment]
Thank you!

**Pull request merged**